### DISCUSSION TITLE
v3.10.0

### DISCUSSION THREAD

**Discussion by rabilrbl** [rabilrbl/jiotv_go#438] [Releases]
## What's Changed
* Use JioTV Go version from VERSION file by @itsyourap in https://github.com/JioTV-Go/jiotv_go/pull/434
* chore(deps-dev): Bump tailwindcss from 3.4.11 to 3.4.12 in /web by @dependabot in https://github.com/JioTV-Go/jiotv_go/pull/435
* chore: switch names and url to jiotv-go org by @rabilrbl in https://github.com/JioTV-Go/jiotv_go/pull/436

**Full Changelog**: https://github.com/JioTV-Go/jiotv_go/compare/v3.9.5...v3.10.0

<hr /><em>This discussion was created from the release <a href='https://github.com/JioTV-Go/jiotv_go/releases/tag/v3.10.0'>v3.10.0</a>.</em>

**Comment by Hitaishi18** [comment]
Which file need to download for firetv stick?

**Reply by rabilrbl** [reply]
> Which file need to download for firetv stick?
> 

Not sure.
Try android5
